wiki/Article_Style_Guide/en.md

```diff
@@ -1,192 +1,318 @@
-4.  Definite and indefinite articles (the, a, and an) should not be placed at the beginning of titles unless they are part of a proper name.
+```md
+# heading level 1
 
-Writing
--------
+## heading level 2
+```
 
-1.  As this wiki's purpose is to document facts, you should always avoid speculative and unsourced information. Generally speaking, information does not require sources if they can directly be seen in-game or are otherwise obvious. Other information however, such as information that is not widely known, must be sourced with a proper reference. Do *not* add content to an article if you cannot find a proper source.
-    1.  This does not applies to guides and tutorials which do not document facts but are here to help the userbase get a better understanding of the game mechanics and are, therefore, subjective. This does not mean that the content of guides and tutorials should not be verified.
+You can choose either the underline style or hashtag style of level 1 and level 2 headings.
+Note that heading levels 3 and onwards use the hashtag style.
 
-2.  Articles in the main namespace should always be written in the third-person perspective and without terms referential to the reader. Try not to use abbreviations of words either. For instance, sentences like "If you miss an object when playing a beatmap, you will break your combo.." should be written as "Failing to hit an object while playing a beatmap will break the current combo.".
+### Article Titles
 
-### Keeping articles concise and up to date
+Titles are to be in title case.
+The title of an article is the name of the folder that the article is located in.
+For English article titles, if you need to reword the title, you must rename the folder to match the article title.
 
-Articles should only contain information that is up to date, *i.e.*, implemented in the latest version of the game. Anything that is outdated should be moved to the History section of the article. When something changes, note the change in the History section and remove the outdated information from other sections of the article. It is unnecessary to mention when a particular feature was implemented; this is once again reserved for the History section of the article.
+All article titles are to be using the level one heading.
 
-### Grammar
+### Section Headings
 
-Pages on the wiki should only use *British English* as a matter of consistency. For instance, “color” should be “colour”, “center” should be “centre”, “skillful” should be “skilful”, “paralleled” should be “parallelled”, and “analyze” should be “analyse”. It is recommended to get accustomed to British English and American English spelling differences before writing an article.
+All section headings are to use title case, just like the article titles.
+
+Section headings must **never** have a link in it.
+Instead place the link underneath the section heading.
+For example:
+
+```
+## Editor
+
+For a full explaination, see [Editor](/wiki/Editor/).
```

Comment:
Nitpicking ze link.

`For a full explanation, see [Beatmap_Editor](/wiki/Beatmap_Editor/).`